docs/comparison-of-builders.md

```diff
@@ -0,0 +1,221 @@
+# Comparison of DiCy to other LaTeX Builders
+
+There are many different automatic and semi-automatic builders for LaTeX-centric
+documents. The [Compilation Topic][] at CTAN provides a current list of some of
+those specifically designed for TeX or LaTeX documents. In addition there are
+also generic document processing tools such as [pandoc][].
+
+In order to explain the intention of and capabilities of DiCy this document
+compares DiCy to two of the most popular builders, [arara][] and [latexmk][].
+
+## arara
+
+[arara][] is a rule based builder which relies on directives given in the source
+file to specify what programs should be run, the order in which they should be
+run, and also the frequency in which they should be run. For instance, to
+compile a document that has a BibTeX based bibliography the following directives
+would need to be listed at the beginning of the source document.
+
+```latex
+% arara: pdflatex
+% arara: bibtex
+% arara: pdflatex
+% arara: pdflatex
+\documentclass{article}
+...
+```
+
+Any repeated calls of `pdflatex` need to be listed explicitly as [arara][] does
+not detect the need to run or rerun a program, nor does it detect which rules
```

Comment:
I'm afraid but I don't understand what "nor does it detect which rules are applicable to the source file" means.